assets/large_language_models/dbcopilot/environments/dbcopilot_embeddings/context/Dockerfile

```diff
@@ -4,8 +4,8 @@ WORKDIR /
 USER root
 
 # Install necessary tools
-RUN apt-get update && apt-get install -y \
-    curl \
+RUN apt-get update && apt-get install -y --allow-downgrades \
+    curl=7.68.0-1ubuntu2.24 \
```

Comment:
all this should come from the latest promtflow image, please wipe unnecessary installs